test/util.go

```diff
@@ -1,10 +1,53 @@
 package test
 
 import (
+	"fmt"
+	"image"
+	"image/draw"
+	"image/png"
+	"os"
+	"path/filepath"
+	"testing"
+
 	"fyne.io/fyne"
 	"fyne.io/fyne/internal/cache"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
+// AssertImagesEqual asserts that the master image at given filename (below testdata) and the given image are equal.
```

Comment:
This documentation needs more information with regards to the filename. It should specify that it is loaded from testdata relative to the test - but more importantly the final part "written to disk" should probably specify location as well.